packages/devtools_app/lib/src/logging/flutter/logging_screen.dart

```diff
@@ -53,16 +61,73 @@ class _LoggingScreenState extends State<LoggingScreenBody> {
         mainAxisAlignment: MainAxisAlignment.spaceBetween,
         children: [
           RaisedButton(
-            onPressed: _clearLogs,
             child: const Text('Clear logs'),
+            onPressed: _clearLogs,
           ),
           StructuredErrorsToggle(),
         ],
       ),
+      Expanded(
+        child: LogsTable(
+          data: controller.data,
+        ),
+      ),
     ]);
   }
 
   void _clearLogs() {
-    // TODO(https://github.com/flutter/devtools/issues/1286): do this.
+    setState(() {
+      controller.clear();
+    });
   }
 }
+
+class LogsTable extends StatefulWidget {
+  const LogsTable({Key key, this.data}) : super(key: key);
+  final List<LogData> data;
+
+  @override
+  _LogsTableState createState() => _LogsTableState();
+}
+
+class _LogsTableState extends State<LogsTable> {
+  final List<ColumnData<LogData>> columns = [
+    FlutterWhenColumn(),
+    FlutterKindColumn(),
+    FlutterMessageColumn((message) => message),
```

Comment:
Are these "Flutter" prefixes to avoid some class name collisions? If so, please add a TODO to remove them once we strip the dart:html app out. If not, please remove in favor or simpler class names.

Reply:
Also feel free to submit a quick CL that renames any classes from the dart:html version of the app to add an Html prefixes. In general I recommend trying to keep the new Flutter version of the codebase clean while making the dart:html version uglier as needed.

Reply:
@kenzieschmoll yes, that's correct.  I've added a TODO to merge these with the parent column classes.

@jacob314 I shortened these names.